backend/src/main/java/com/crm/model/db/AddressEntity.java

```diff
@@ -0,0 +1,56 @@
+package com.crm.model.db;
+
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.validation.constraints.Max;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.Size;
+
+@Entity
+@NoArgsConstructor
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
+@Getter
+@Setter
+@EqualsAndHashCode
+@Builder
+@Table(name = "addresses")
+public class AddressEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @NotBlank
+    @Size(max = 30)
+    private String country;
+
+    @NotBlank
+    @Size(max = 30)
+    private String city;
+
+    @NotBlank
+    @Size(max = 30)
+    private String postalCode;
+
+    @NotBlank
+    @Size(max = 30)
```

Comment:
The longest street name in poland has 56 characters. Maybe consider this in size of street name.
https://tvn24.pl/polska/tna-nazwy-ulic-z-oszczednosci-ra292263-3515213

Reply:
Haha, ok I will change that